daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1049,17 +1042,15 @@ object Main extends Logging {
                         "at least " + (bytesAvailable * 8)
                       }
                     val leftOverDataWarning = s"Left over data. Consumed ${loc.bitPos0b} bit(s) with ${remainingBits} bit(s) remaining." + firstByteString + dataHex + dataText
```

Comment:
This is no longer a warning, so this variable name isn't quite right. Maybe we should rename it `leftOverDataMessage`?